### PR TITLE
Issue #3305: info message style affects "Available Updates" page.

### PR DESCRIPTION
**Pull request by klonos**
https://github.com/backdrop/backdrop-issues/issues/3305

### CODE REVIEW

**Comment by klonos** [comment]
...closing and re-opening the PR in order to fix "Failed to get code from github." error...

**Comment by klonos** [comment]
...reopening

File: core/modules/update/update.theme.inc [hunk 1]
```diff
@@ -272,7 +273,7 @@ function theme_update_report($variables) {
         $sub_section .= "</div>\n";  // .subthemes div
       }
 
-      $info_section = '<div class="info">' . $extra_section . $includes_section . $base_section . $sub_section . '</div>';
+      $info_section = '<div class="project-info">' . $extra_section . $includes_section . $base_section . $sub_section . '</div>';
```
Comment (opi) [inline core/modules/update/update.theme.inc:276]: Changes needed in `core/modules/update/css/update.css`

- https://github.com/backdrop/backdrop/blob/3d5074a261c5af713286614d1aa2048919e69b00/core/modules/update/css/update.css#L36 
- https://github.com/backdrop/backdrop/blob/3d5074a261c5af713286614d1aa2048919e69b00/core/modules/update/css/update.css#L127-L134

File: core/modules/system/css/messages.theme.css [hunk 2]
```diff
@@ -56,50 +56,50 @@
...
+.messages.warning {
   color: #525252;
   background-color: #faf5c9;
 }
-.warning a {
+.messages.warning a {
   color: #525252;
 }
-div.warning:before {
+div.messages.warning:before {
```
Comment (opi) [inline core/modules/system/css/messages.theme.css:78]: Don't know why the `:before` rules need a `div` tag in selector... probably some cascading reason.

**Review by opi — CHANGES_REQUESTED** [review]

File: core/modules/update/update.theme.inc (hunk 1 above)
Comment (klonos) [reply core/modules/update/update.theme.inc:276]: Good catch @opi 👍. Updated PR.

File: core/modules/system/css/messages.theme.css (hunk 2 above)
Comment (klonos) [reply core/modules/system/css/messages.theme.css:78]: Don't know either @opi. I imagine specificity wars at some point. I chose to keep what we already had in place. so not to break things.

Comment (opi) [reply core/modules/system/css/messages.theme.css:78]: agreed

**Review by opi — APPROVED** [review]

**Comment by herbdool** [comment]
Thank you! Merged into 1.x and 1.11.x.

**Pull request merged**